alicloud/table_alicloud_ram_user.go

```diff
@@ -36,9 +36,11 @@ func tableAlicloudRAMUser(ctx context.Context) *plugin.Table {
 			Hydrate: listRAMUser,
 		},
 		Get: &plugin.GetConfig{
-			KeyColumns:        plugin.SingleColumn("name"),
-			ShouldIgnoreError: isNotFoundError([]string{"EntityNotExist.User", "MissingParameter"}),
-			Hydrate:           getRAMUser,
+			KeyColumns: plugin.SingleColumn("name"),
+			IgnoreConfig: &plugin.IgnoreConfig{
+				ShouldIgnoreErrorFunc: isNotFoundError([]string{"EntityNotExist.User", "MissingParameter"}),
```

Comment:
Do we need to add the `EntityNotExist.User` error code here?
We have already handled this in [DefaultGetConfig](https://github.com/turbot/steampipe-plugin-alicloud/pull/344/files#diff-5f561e6fa39974710222f0ead49a126eec949864b6c87646c290c36cc88db682R17) for this plugin.